src/nvim/eval.c

```diff
@@ -2862,8 +2862,10 @@ static int eval5(char **arg, typval_T *rettv, evalarg_T *const evalarg)
         }
       } else {
         bool error = false;
-        varnumber_T n1, n2;
-        float_T f1 = 0, f2 = 0;
+        varnumber_T n1;
+        varnumber_T n2;
```

Comment:
Weren't those "GOOD" before?

Reply:
They were, I just couldn't be bothered to adjust after clang-tidy fixed it. I can change back.

Reply:
Just wondering, but if you want to keep them, I'd suggest explicitly documenting that as "GOOD" as well.